.github/workflows/assert.sh/assert.sh

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
+
```

Comment:
Was this file taken from somewhere?
I found something similar https://github.com/torokmark/assert.sh/blob/main/assert.sh

If it is external then may be add link comment for where it was taken from or may be add that to somewhere in README.md??

I have reviewed the file thinking you created it so ignore all the below review comments for the file if it was taken externally and I will resolve. Apologies for that!